public/app/features/explore/SupplementaryResultError.tsx

```diff
@@ -19,27 +20,52 @@ export function SupplementaryResultError(props: Props) {
   // /public/app/features/explore/ErrorContainer.tsx
   const message = error?.message || error?.data?.message || '';
   const showButton = !isOpen && message.length > SHORT_ERROR_MESSAGE_LIMIT;
+  const theme = useTheme2();
+  const styles = getStyles(theme);
 
   return (
-    <Alert title={title} severity={severity} onRemove={onRemove}>
-      {showButton ? (
-        <Button
-          variant="secondary"
-          size="xs"
-          onClick={() => {
-            setIsOpen(true);
-          }}
-        >
-          Show details
-        </Button>
-      ) : (
-        message
-      )}
-      {suggestedAction && onSuggestedAction && (
-        <Button variant="primary" size="xs" onClick={onSuggestedAction}>
-          {suggestedAction}
-        </Button>
-      )}
-    </Alert>
+    <div className={styles.supplementaryErrorContainer}>
+      <Alert title={title} severity={severity} onRemove={onRemove}>
+        <div className={styles.suggestedActionWrapper}>
+          {showButton ? (
+            <Button
+              variant="secondary"
+              size="xs"
+              onClick={() => {
+                setIsOpen(true);
+              }}
+            >
+              Show details
+            </Button>
+          ) : (
+            message
+          )}
+          {suggestedAction && onSuggestedAction && (
+            <div className={styles.suggestedActionWrapper}>
+              <Button variant="primary" size="xs" onClick={onSuggestedAction}>
+                {suggestedAction}
+              </Button>
+            </div>
+          )}
+        </div>
+      </Alert>
+    </div>
   );
 }
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
```

Comment:
These styles are necessary because the `Alert` component has `overflow: auto` set for the content. https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Alert/Alert.tsx#L168

![Overflow](https://user-images.githubusercontent.com/1069378/229186599-a34cb556-adda-4205-9d64-d2906a0d4ba7.png)